Fix alpine docker image error

jtomass opened the pull request:
Change FROM to a ready built docker image

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vaadin/framework/10382)
<!-- Reviewable:end -->

jtomass committed:
Fix alpine docker image error

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/vaadin/framework?pullRequest=10382) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/vaadin/framework?pullRequest=10382) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/vaadin/framework?pullRequest=10382) it.</sub>

tsuoanttila commented:
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[scripts/BuildHelpers.py, line 147 at r1](https://reviewable.io:443/reviews/vaadin/framework/10382#-L-1unU9AFxvqrYsXShm:-L-1unU9AFxvqrYsXShn:b3qz4sh) ([raw file](https://github.com/vaadin/framework/blob/5743b2f7babebd5cc683426e3a3dac7b36928dbf/scripts/BuildHelpers.py#L147)):*
> ```Python
> 	dockerFileContent = """FROM jtomass/alpine-jre-bash:latest
> MAINTAINER FrameworkTeam
> ```

Multiple empty lines

---

*[scripts/BuildHelpers.py, line 151 at r1](https://reviewable.io:443/reviews/vaadin/framework/10382#-L-1ub_lysP7YaOzNVNT:-L-1ub_lysP7YaOzNVNU:b-a88mge) ([raw file](https://github.com/vaadin/framework/blob/5743b2f7babebd5cc683426e3a3dac7b36928dbf/scripts/BuildHelpers.py#L151)):*
> ```Python
> COPY ./*.war /var/lib/jetty/webapps/
> COPY ./index-generate.sh /opt/
> RUN chmod +x /opt/index-generate.sh
> ```

Running this line when generating the new image threw `Operation not permitted`

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/10382)*
<!-- Sent from Reviewable.io -->

tsuoanttila commented:
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/10382)*
<!-- Sent from Reviewable.io -->

jtomass committed:
Alpine linux user right fix

Base of the problem presented in the original problem was alpine linux different USER rights and also in the first commits error.

tsuoanttila commented:
Tested and <img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/10382#-:-L-5zzblmxrimTq_Wsdh:b-unr4qp)*
<!-- Sent from Reviewable.io -->
